apps/web/pages/api/download-cal-video-recording.ts

```diff
@@ -25,7 +26,7 @@ async function handler(
   const session = await getSession({ req });
 
   //   Check if user belong to active team
-  if (!session?.user?.belongsToActiveTeam) {
+  if (!session?.user?.belongsToActiveTeam && !IS_SELF_HOSTED) {
```

Comment:
I think following is much more readable.
```suggestion
const isDownloadAllowed = IS_SELF_HOSTED ? true : !session?.user?.belongsToActiveTeam`
if (!isDownloadAllowed) {
```